apps/endatix-hub/app/(main)/forms/ui/create-form-sheet.tsx

```diff
@@ -80,10 +81,11 @@ const CreateFormSheet = () => {
             formDefinitionJsonData: JSON.stringify("{ }")
         }
         const formResult = await createFormAction(request);
-        if (formResult.isSuccess && formResult.formId) {
-            redirect(`/forms/${formResult.formId}`);
+        if (Result.isSuccess(formResult) && formResult.value) {
```

Comment:
Here idea is you can just say `Result.isSuccess()`, but I see that's not guaranteed yet. I need to change the API to make the value check redundant.